src/app.js

```diff
@@ -86,7 +86,13 @@ export const start = args => {
   };
 
   screen.key('?', toggleHelp);
-  screen.key(['escape', 'q', 'C-c'], () => process.exit(0));
+  screen.key(['escape', 'q', 'C-c'], () => {
+    if (screen.lockKeys) {
+      killYoungestChild();
+    } else {
+      process.exit(0);
+    }
+  });
   screen.key('C-r', () => {
     branchTable.clearItems();
```

Comment:
Ah https://github.com/jwu910/check-it-out/pull/262/files#r222372170 this comment is referring to this line. 

It might actually be safe to remove this line, and keep the one in the promise .then() becuase we if we clear here, and `doFetchBranches()` fails, we may not even want to clear our list yet.

I'll open an issue to clean this up. let's leave it for out as it's out of this ticket scope.

Reply:
Removing the first clearItems() causes the loading screen to overlap with branchTable. You end up getting something like this:

```
Fetching refs...

/
    heads 6.1.x
```

Should I leave in both clearItems until the issue is resolve later?

Reply:
Lol that's probably why I had it in there in the first place :+1:

Reply:
@jwu910 I also made one additional change in these commits. 

I changed gitResponse.on('close') to 'exit'. 'exit' seems to throw SIGTERM immediately when the user presses q. 'close' waits for the process streams to close. Since the pid of gitResponse is the same as cio, and the cio streams don't close until the program exits, the child process won't actually terminate at all, and will proceed to execute the command.